drivers/sensor/lsm6dsl/Kconfig

```diff
@@ -8,13 +8,30 @@
 #
 
 menuconfig LSM6DSL
-	bool "LSM6DSL I2C accelerometer and gyroscope Chip"
-	depends on I2C
+	bool "LSM6DSL I2C/SPI accelerometer and gyroscope Chip"
+	depends on SENSOR && (I2C || SPI)
 	default n
 	help
-	  Enable driver for LSM6DSL I2C-based accelerometer and gyroscope
+	  Enable driver for LSM6DSL accelerometer and gyroscope
 	  sensor.
 
+choice
+	prompt "Interface to AP"
+	depends on LSM6DSL
+	default LSM6DSL_SPI
```

Comment:
Default should be _I2C, esle we'll have to rework all other boards that are using I2C today (by default) and which don't specify it